aspnetcore/mvc/views/view-components.md

```diff
@@ -82,6 +82,14 @@ The default view name for a view component is *Default*, which means your view f
 
 We recommend you name the view file *Default.cshtml* and use the *Views/Shared/Components/{View Component Name}/{View Name}* path. The `PriorityList` view component used in this sample uses *Views/Shared/Components/PriorityList/Default.cshtml* for the view component view.
 
+### Customizing the view search path
+
+To customize the view search path, modify Razor's `ViewLocationFormats` collection. For example, to search for views within the path "/Components/{View Component Name}/{View Name}", add a new item to the collection:
+
+[!code-cshtml[](view-components/samples_snapshot/2.x/Startup.cs?name=snippet_ViewLocationFormats&highlight=4)]
```

Comment:
```suggestion
[!code-cs[](view-components/samples_snapshot/2.x/Startup.cs?name=snippet_ViewLocationFormats&highlight=4)]
```

Reply:
Thanks @scottaddie. I copied that from above to make sure I got it right and still I didn't. :smile:

Reply:
The following is a useful resource when determining what to use: https://github.com/highlightjs/highlight.js#supported-languages.